qcodes/instrument_drivers/QDev/QDac.py

```diff
@@ -30,8 +31,9 @@ class QDac(VisaInstrument):
 
     # set nonzero value (seconds) to accept older status when reading settings
     max_status_age = 1
-
+    
     def __init__(self, name, address, num_chans=48, update_currents=True):
+        warnings.warn("The QDac Class is deprecated; use QDacChannel instead.", DeprecationWarning, 2)
```

Comment:
I don't mean to nitpick, but the `QDac` class from `QDac.py` is being deprecated in favour of the `QDac` class from `QDac_channels.py`. The module `QDac_channels.py` also contains a class called `QDacChannel`, but that is not an instrument driver; the driver is called `QDac`.